core/src/cats/sql.cc

```diff
@@ -526,7 +526,7 @@ int BareosDb::ListResult(void* vctx, int, char** row)
         } else {
           value.bsprintf("%s", row[i]);
         }
-        send->ObjectKeyValue(field->name, value.c_str(), " %s");
+        send->ObjectKeyValue(field->name, value.c_str(), "%s");
```

Comment:
Im not sure about this, but it should be fine.

Reply:
Agreed. Removed again.